sparse/coo.py

```diff
@@ -712,13 +712,15 @@ def reduce(self, method, axis=(0,), keepdims=False, **kwargs):
         if not isinstance(axis, tuple):
             axis = (axis,)
 
+        axis = tuple(a if a >= 0 else a + self.ndim for a in axis)
```

Comment:
Are there other uses of `axis=` where this should be done but is not yet currently done?

Reply:
With a short "find all" of `axis`, I'm inclined to say no. It does appear in `tensordot`, `concatenate` and `stack` but those already have this.